### DISCUSSION TITLE
Realtime inference doesn't work..

### DISCUSSION THREAD

**Discussion by F-V-Younesi** [voicepaw/so-vits-svc-fork#936] [Q&A]
Realtime inference doesn't work in colab.
when I ran "!svc vc" for realtime inference, this error appears:
W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
[11:02:34] INFO     [11:02:34] NumExpr defaulting to 2 threads.     

Should I use another command? If not, how can I fix this error?

GPU: NVIDIA Tesla T4 
chrome :Version 116.0.5845.180 (Official Build) (32-bit)

**Comment by 34j** [comment]
Duplicate and wontfix
